style.css

```diff
@@ -185,3 +182,389 @@ a:hover {
+}
+
+.sup {
+  width: 327px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  color: #091e42;
+  position: relative;
+  bottom: 40px;
+  margin-top: 50px;
+}
+
+.textblock-tags {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  width: 297px;
+  height: 40px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  align-self: baseline;
+  margin-top: -158px;
+  list-style-type: none;
+  margin-left: 24px;
+}
+
+.textblock-tags > li {
+  background: #fff;
+  border: 1px solid #8993a4;
+  padding: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.textblock-link {
+  background-color: #ff6b00;
+  color: white;
+  padding: 14px 25px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  bottom: 30px;
+  width: 124px;
+}
+
+#project {
+  order: 5;
+  border-radius: 8px;
+}
+
+.proj {
+  display: flex;
+  flex-direction: row;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  list-style-type: none;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  margin-top: 0;
+  width: 200px;
+  height: 40px;
+  margin-left: 24px;
+}
+
+.frame43 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 24px;
+  width: 375px;
+  height: 2436px;
+  position: relative;
+  left: 24px;
+  margin-top: 12px;
+}
+
+.content-defaults {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 386px;
+  background: url('../Images/Img-Placeholder.png') no-repeat;
+  background-color: #f4f5f7;
+}
+
+.other-content-title {
+  width: 295px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  padding-left: 12px;
+  color: #fff;
+  position: relative;
+  top: 50px;
+}
+
+.other-content-description {
+  width: 295px;
+  height: 87px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-left: 12px;
+  color: #fff;
+  position: relative;
+  top: 10px;
+}
+
+.other-content-tags {
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+  width: 236px;
+  height: 40px;
+  list-style-type: none;
+  color: #fff;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  box-sizing: border-box;
+  position: relative;
+  right: 30px;
+}
+
+.other-content-tags > li {
+  width: fit-content;
+  padding: 10px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.other-content-link {
+  background-color: #ff6b00;
+  color: white;
+  padding: 14px 25px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  position: relative;
+  top: 10px;
+}
+
+.about {
+  position: absolute;
+  width: 375px;
+  height: 1601px;
+  left: 0;
+  top: 4000px;
+  background: url("../Images/Frame.png") no-repeat;
```

Comment:
[OPTIONAL] **There is an extra dot `.` in your image URL kindly make sure you keep one dot. 
Your code should look like this :arrow_down:**

` list-style-image: url("./Images/Rectangle55.png");`  **Notice that I removed one dot from the URL path**